portality/models/oaipmh.py

```diff
@@ -93,23 +95,26 @@ def list_records(self, from_date=None, until_date=None, oai_set=None, list_size=
         if list_size is not None:
             q["size"] = list_size
 
-        if start_number is not None:
-            q["from"] = start_number
+        if start_after is not None:
+            q["from"] = start_after[1]
+        else:
+            q["from"] = 0
 
-        q["sort"] = [deepcopy(self.created_sort)]
+        q["sort"] = deepcopy(self.created_sort)
 
         # do the query
         # print json.dumps(q)
+
         results = self.query(q=q)
 
         total = results.get("hits", {}).get("total", 0)
         return total, [hit.get("_source") for hit in results.get("hits", {}).get("hits", [])]
 
 
 class OAIPMHArticle(OAIPMHRecord, Article):
-    def list_records(self, from_date=None, until_date=None, oai_set=None, list_size=None, start_number=None):
+    def list_records(self, from_date=None, until_date=None, oai_set=None, list_size=None, start_number=None, start_after=None):
         total, results = super(OAIPMHArticle, self).list_records(from_date=from_date,
-            until_date=until_date, oai_set=oai_set, list_size=list_size, start_number=start_number)
+            until_date=until_date, oai_set=oai_set, list_size=list_size, start_number=start_number, start_after=start_after)
```

Comment:
Same comment as above: start_number can be removed from the method signature, and doesn't need to be passed forward here.

Reply:
same answer as above :)

Reply:
Fixed here too